src/styles/app/04-sidebar.css

```
@@ -277,6 +278,48 @@
+}
+
+.sidebarStackHeaderSortNative:hover,
+.sidebarStackHeaderSortNative:focus-within {
+	color: var(--text-primary);
+}
+
+.sidebarStackHeaderSortIcon {
+	position: absolute;
+	left: 4px;
+	pointer-events: none;
+	color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)